qemu/tests/hugepage_specify_node.py

```diff
@@ -63,24 +66,35 @@ def run(test, params, env):
                                   idle_node, test.log.info)
         params["setup_hugepages"] = "yes"
         hp_config = test_setup.HugePageConfig(params)
-        hp_config.setup()
-        params["qemu_command_prefix"] = "numactl --membind=%s" % node_id
-        params["start_vm"] = "yes"
-        params["hugepage_path"] = hp_config.hugepage_path
-        env_process.preprocess_vm(test, params, env, params["main_vm"])
         try:
+            hp_config.setup()
+            params["qemu_command_prefix"] = "numactl --membind=%s" % node_id
+            params["start_vm"] = "yes"
+            params["hugepage_path"] = hp_config.hugepage_path
+            env_process.preprocess_vm(test, params, env, params["main_vm"])
```

Comment:
The `preprocess_vm`  does not handle hugepage setup while `preprocess` do this.   So I did not see any meaningful call for `params["setup_hugepage"] =  yes`.

I am ok with both of the following two solutions:
1. setup/cleanup hugepage in testcase and use `preprocess_vm`
2. set `setup_hugepage = yes`, use `preprocess` to make framework help handle the setup and cleanup automatically.

Reply:
Thanks a lot, it has been corrected.
Remove `params["setup_hugepage"] = yes` 
because you need to setup hugepage multiple times in the environment.